# This message contains the state vector [x y z phi theta psi u_x u_y u_z w_phi w_theta w_psi]^T

Header header

float64 x
float64 y
float64 z
float64 phi
float64 theta
float64 psi
float64 u_x
float64 u_y
float64 u_z
float64 w_phi
float64 w_theta
float64 w_psi
